includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1267,6 +1267,11 @@ public function process_payment_for_order( $cart, $payment_information, $additio
 		if ( $payment_needed ) {
 			$payment_method_details = $intent->get_payment_method_details();
 			$payment_method_type    = $payment_method_details ? $payment_method_details['type'] : null;
+
+			if ( $payment_method_details && 'card' === $payment_method_details['type'] ) {
+				$order->add_meta_data( 'last4', $payment_method_details['card']['last4'] );
```

Comment:
This works fine when I check out this PR only on the WooPay site, but if I check it out on _both_ the WooPay site and on the merchant site, then `last4` ends up appearing twice in the metadata with the same value — I assume this is happening because it's first coming from this payment data and it's also getting added by WCPay on the merchant site. Can we make sure the order meta only gets added once?